version.sh

```diff
@@ -46,7 +46,7 @@ update_file_version()
     NEW_LINE=$3
 
     echo "Update $FILE [$LINE_PATTERN] => [$NEW_LINE]"
-    sed -i s/"$LINE_PATTERN"/"$NEW_LINE"/g $FILE
+    sed -i ''  s/"$LINE_PATTERN"/"$NEW_LINE"/g $FILE
```

Comment:
These sed changes cause it to not work on GNU. I think requesting that Mac OS users install GNU sed via `brew install gnu-sed` may be the best bet unless @bfjelds has any idea as to how to make the sed magic happen.

Reply:
Okay let me try later with `gnu-sed`